rules/framework/framework_packaging.py

```diff
@@ -81,7 +81,11 @@ def _clean(framework_root, manifest_file, output_manifest_file):
         for d in dirs:
             path = os.path.join(root, d)
             if path not in dirs_to_keep:
-                shutil.rmtree(path)
+                if os.path.islink(path):
```

Comment:
Is this code impacting you BTW - we've got another PR that remobes it altogether. You can set `--features apple.virtualize_frameworks` - in a bigger build with thousands of deps rules_ios is pretty much useable from a performance perspective

Reply:
My real project has enabled `virtualize_frameworks`, I only hit on this when running the rules_ios tests.